code/game/objects/items/weapons/handcuffs.dm

```diff
@@ -158,3 +158,93 @@ var/last_chew = 0
 
 /obj/item/weapon/handcuffs/cyborg
 	dispenser = 1
+
+
+/obj/item/weapon/handcuffs/ziptie
+	name = "zipties"
+	desc = "Sturdy, and reliable plastic zipties for binding the wrists."
+	gender = PLURAL
+	icon = 'icons/obj/items.dmi'
+	breakouttime = 600 //One minute, because they're not made of steel, therefore easier to slip out of.
+	icon_state = "ziptie"
+	cuff_sound = 'sound/weapons/cablecuff.ogg'
+	var/singular_name = "ziptie"
+	var/amount = 6
+	var/max_amount = 6
+
+/obj/item/weapon/handcuffs/ziptie/New(var/loc, var/amount=null)
+	..()
+	if (amount)
+		src.amount=amount
+	return
+
+/obj/item/weapon/handcuffs/ziptie/Del()
+	if (src && usr && usr.machine==src)
+		usr << browse(null, "window=stack")
+	..()
+
+/obj/item/weapon/handcuffs/ziptie/examine()
+	set src in view(1)
+	..()
+	usr << "There are [src.amount] [src.singular_name]\s in the stack."
+	return
+
+/obj/item/weapon/handcuffs/ziptie/proc/use(var/amount)
+	src.amount-=amount
+	if (src.amount<=0)
+		src = null //dont kill proc after del()
+	return
+
+/obj/item/weapon/handcuffs/ziptie/proc/add_to_stacks(mob/usr as mob)
+	var/obj/item/weapon/handcuffs/ziptie/oldsrc = src
+	src = null
+	for (var/obj/item/weapon/handcuffs/ziptie/N in usr.loc)
+		if (N==oldsrc)
+			continue
+		if (!istype(N, oldsrc.type))
+			continue
+		if (N.amount>=N.max_amount)
+			continue
+		oldsrc.attackby(N, usr)
+		usr << "You add new [N.singular_name] to the stack. It now contains [N.amount] [N.singular_name]\s."
+		if(!oldsrc)
+			break
+
+/obj/item/weapon/handcuffs/ziptie/attack_hand(mob/user as mob)
+	if (user.get_inactive_hand() == src)

```

Comment:
Bad behaviour will emerge from this. If you've got one ziptie in hand, it'll delete that one, and make a new one in the other hand. Which is silly.

Change this if to, `if (user.get_inactive_hand() == src && amount > 1)` maybe? This way, default behaviour will initiate if the stack consists of one tie.